indico_toolkit/auto_populate/populator.py

```diff
@@ -211,7 +212,50 @@ def copy_teach_task(
             raise ToolkitPopulationError("Error: Failed to submit labels")
         return workflow
 
-    def _get_labels_by_filename(
+    def inject_labels_into_teach_task(
+        self,
+        workflow_id: int,
+        teach_task_id: int,
+        file_to_targets: dict,
+        rename_labels: Dict[str, str] = None,
+        remove_labels: List[str] = None
+    ):
```

Comment:
add the return object type hint (if we're retuning) or remove it

Reply:
removed